java/src/jmri/server/json/throttle/JsonThrottle.java

```diff
@@ -464,7 +450,7 @@ private ObjectNode getStatus() {
         data.put(Throttle.F26, this.throttle.getF26());
         data.put(Throttle.F27, this.throttle.getF27());
         data.put(Throttle.F28, this.throttle.getF28());
-        data.put(SPEED_STEPS, this.speedSteps);
+        data.put(SPEED_STEPS, this.speedSteps.name);
```

Comment:
`java/src/jmri/server/json/throttle/throttle-client.json` and `java/src/jmri/server/json/throttle/throttle-server.json` represent a contract between the JMRI developers and third-party developers that the JMRI JSON server will behave in a certain way. If you are going to change that contract, update `java/src/jmri/server/json/packge-info.java` as well.

Reply:
Ok; I've added a release note to `java/src/jmri/server/json/package-info.java` and marked the new version as 5.1.0 for JMRI 4.17.2 (I think that will be the next release). Please let me know if either of those versions need to be changed.

Reply:
On second review, I'm going to ask that you undo the changes you did to package-info.java, thottle-client.json, and throttle-server.json. The JsonThrottle is *not* telling the client the speed step mode, it is telling the client the number of speed steps (note that original line 371 has the value "126", not the value "128" for the NMRA DCC 128 speed step mode).

Reply:
Good catch.